idl/query.idl.hh

```diff
@@ -27,7 +27,6 @@ class qr_partition stub [[writable]] {
 
 class query_result stub [[writable]] {
     utils::chunked_vector<qr_partition> partitions; // in ring order
-    std::optional<full_position> last_position [[version 5.1]];
 };
```

Comment:
Please extract this commit into its own pull so we can commit it quickly and not accidentally cut a branch while this is making progress.